CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ IF(WITH_CORE)
   FOREACH (GRASS_SEARCH_VERSION 7)
     # Legacy note:
     # For GRASS 6 there were used cached variables without version suffix so that existing caches didn't have to be reconfigured.
-    # Cached variables were: WITH_GRASS, WITH_GRASS7, GRASS_PREFIX, GRASS_PREFIX7, GRASS_INCLUDE_DIR, GRASS_INCLUDE_DIR7
+    # Cached variables were: WITH_GRASS, GRASS_PREFIX, GRASS_INCLUDE_DIR, GRASS_INCLUDE_DIR7
```

Comment:
Yes, WITH_GRASS7, GRASS_PREFIX7 and GRASS_INCLUDE_DIR7 are used and those are the only GRASS options. I decided to keep "7" suffix so that build scripts do not have to be modified.